### ISSUE TITLE
Realism-1.4.2-RC1: AI can not discharge weapons

### ISSUE THREAD

**Issue by uamadman** [space-commits/SPT-Realism-Mod-Client#81]
This is on a fresh install of 3.9.5 and a fresh server.

The AI was fully functional with just SPT and Fika, but the addition of realism broke it.
The AI has ammo on their person, is able to reload, but can not discharge their weapons. 
They can reload forever and if i remember correctly they don't consume ammo on their bodies? I can't remember.
Edit:  After standing next to an AI for 15 minutes they stopped reloading and had no ammo on their bodies.

![Server-state](https://github.com/user-attachments/assets/47a1e2e7-7fb9-4860-b23c-946a76f7f705)
![Client](https://github.com/user-attachments/assets/8fcf546c-42c4-4a64-8db1-43a1dd21dc50)
![PMC-Reloading](https://github.com/user-attachments/assets/c5ba2998-f135-47b0-a52b-57d4f7bd790f)
![Realism-kit-1](https://github.com/user-attachments/assets/2814ca9a-da29-470a-b948-bad3c2031494)
![server](https://github.com/user-attachments/assets/3f98948b-2c32-4bfd-a5dc-0cac4e023bd0)

TLDR; 
Spt 3.9.5 + Fika for 3.9.5 = good
Spt 3.9.5 + Fika for 3.9.5+ Realism 1.4.2 = The AI can no longer discharge their weapons.
Spt 3.9.5 + Fika for 3.9.5 + Realism 1.4.1 + AI level function Patch = good
Edit: Spt 3.9.5 + Realism 1.4.2 = AI Discharge Bug

Client: 
Fika.Release.0.9.8983.25381
Realism-Mod-v1.4.2-SPT-v3.9.5

Linux Server State:
```
┌─────────────────────────────────────────┐
│ SPT 3.9.5                               │
│ https://discord.sp-tarkov.com           │
│                                         │
│ This work is free of charge             │
│ If you paid money, you were scammed     │
│ Commercial use is prohibited            │
└─────────────────────────────────────────┘
ModLoader: loading: 2 server mods...
Mod: SPT Realism Mod version: 1.4.2 by: Fontaine loaded
Mod: server version: 2.2.7 by: Fika loaded
Server: executing startup callbacks...
Importing database...
Database import finished
Started webserver at http://0.0.0.0:6969
```

**Comment by uamadman** [comment]
![image](https://github.com/user-attachments/assets/50db2b46-f11b-4777-ad8b-fbe2992c5489)

After standing in raid, this is the only error that is popping into the logs... doesn't directly seem realism based, but it doesn't exist during normal SPT+Fika runs..

**Comment by space-commits** [comment]
should be resolved in RC2

**Comment by MatthewP098** [comment]
Christmas and Easter are pagan holidays not commanded anywhere in the Bible. They serve other gods and we're breaking the ten commandments when we celebrate them so we gotta repent. If you read Romans 1:22-27, serving other gods is the reason why we're living in Sodom and Gomorrah right now.

Also check Ezekiel 20:19-21, the Sabbath is Saturday morning to Sunday morning and it's the sign between God and his people, so if you don't have that sign God is gonna pour out his fury upon you in tribulation.

And read Isaiah 66:15-17, people who eat unclean like pork and shellfish are going to be consumed by fire in the tribulation. So repent from eating this stuff.

And the Messiah's name isn't Jesus, it's Yeshua, he comes in the Father's name Yehovah. He warned us about another coming in his own name in John 5:42-43 because you don't have the love of God in you (the ten commandments - 1 John 5:2-3).
